### PR TITLE
Improve OKR shortcuts

### PR DESCRIPTION
**Pull request by syh-cpdsss**
## Summary
This PR adds four new OKR shortcuts to streamline common OKR management operations: `+batch-create`, `+reorder`, `+weight`, and `+indicator-update`, improving success rates and reduce token usage for end-to-end OKR workflows.

## Changes
- **`+batch-create` shortcut** (`shortcuts/okr/okr_batch_create.go`): Atomic batch creation of Objectives and Key Results with automatic rollback on failure. Supports `@mention` in content, file/stdin input via `@path`/`@-`, and `--dry-run` for preview.
- **`+reorder` shortcut** (`shortcuts/okr/okr_reorder.go`): Position-based reordering for both Objectives (cycle-level) and Key Results (objective-level). Accepts partial position changes and automatically constructs the full ordered ID list required by the native `objectives_position`/`key_results_position` APIs.
- **`+weight` shortcut** (`shortcuts/okr/okr_weight.go`): Smart weight allocation for Objectives and KRs. Supports partial weight specification with automatic normalization of unspecified items based on original weight proportions, ensuring the total always equals 1.0.
- **`+indicator-update` shortcut** (`shortcuts/okr/okr_indicator_update.go`): Simplified indicator current value update. Automatically looks up the indicator ID for a given Objective/KR ID and updates the `current_value` field, eliminating the need for a separate indicator list API call.
- **Skill documentation** (`skills/lark-okr/SKILL.md`): Added all four shortcuts to the Shortcuts table with links to detailed reference docs.
- **Reference documentation**: Added comprehensive usage docs for each shortcut (`lark-okr-batch-create.md`, `lark-okr-reorder.md`, `lark-okr-weight.md`, `lark-okr-indicator-update.md`) including command examples, parameter tables, input/output formats, and workflow guidance.
- **Unit tests**: Added comprehensive test coverage for all four shortcuts (4594 lines across implementation and test files), covering success paths, edge cases, validation errors, and rollback scenarios.
- **E2E tests**: Added `tests/cli_e2e/okr/okr_shortcuts_test.go` with dry-run and live E2E tests validating the complete shortcut request/response flow.
- **Shortcut registration**: Registered all four new shortcuts in `shortcuts/okr/shortcuts.go`.

## Test Plan
- [x] Unit tests pass
- [x] Manual Test & evaluation pass

## Related Issues
- None

## Release Notes

* **New Features**
  * Added `okr +batch-create` to batch-create OKR objectives and key results from JSON, with validation, optional mentions, dry-run planning, and rollback on failures.
  * Added `okr +indicator-update` to update an indicator’s `current_value` at objective or key-result level, including dry-run support.
  * Added `okr +reorder` to reorder objectives or key results via `--ops`, with dry-run previews.
  * Added `okr +weight` to set objective/key-result weights with validation and normalization to a total of 1.0, plus dry-run.
* **Documentation**
  * Added reference docs and updated the shortcuts table for the new commands.
* **Tests**
  * Added unit and CLI end-to-end coverage for validation, dry-run output, and live behavior (including rollback/error paths).

### CODE REVIEW

File: tests/cli_e2e/okr/okr_shortcuts_test.go [hunk 1]
```diff
@@ -0,0 +1,454 @@
...
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+	t.Cleanup(cancel)
+
+	// Create test objectives
+	created := createTestObjectives(t, ctx, cycleID, suffix)
+
+	// Register cleanup
+	t.Cleanup(func() {
+		cleanupLiveTest(t, created)
+	})
```
[the remark this answers is not in the document]
Comment (syh-cpdsss) [reply tests/cli_e2e/okr/okr_shortcuts_test.go:294]: fixed

File: skills/lark-okr/references/lark-okr-weight.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  "data": {
+    "level": "objective",
+    "cycle_id": "7000000000000000001",
+    "weights": [
+      {"id": "7000000000000000002", "weight": 0.6},
+      {"id": "7000000000000000003", "weight": 0.3},
+      {"id": "7000000000000000004", "weight": 0.1}
+    ]
+  }
+}
```
[the remark this answers is not in the document]
Comment (syh-cpdsss) [reply skills/lark-okr/references/lark-okr-weight.md:84]: fixed

File: skills/lark-okr/references/lark-okr-weight.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+# 调整 KR 权重（全部指定，和为 1）
+lark-cli okr +weight \
+  --cycle-id 7000000000000000001 \
+  --level key-result \
+  --objective-id 7000000000000000002 \
+  --weights '[
+    {"id": "7000000000000000004", "weight": 0.5},
+    {"id": "7000000000000000005", "weight": 0.3}
+  ]' \
+  --as user
```
[the remark this answers is not in the document]
Comment (syh-cpdsss) [reply skills/lark-okr/references/lark-okr-weight.md:29]: fixed

File: skills/lark-okr/references/lark-okr-reorder.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+- 不允许将多个 objective/key-result 放在同一个位置下
+
+## 参数
+
+| 参数               | 必填 | 默认值    | 说明                                                      |
+|------------------|----|--------|---------------------------------------------------------|
+| `--level`        | 是  | —      | 调整层级：`objective`（调整周期下目标顺序）\| `key-result`（调整目标下 KR 顺序） |
+| `--cycle-id`     | 是  | —      | OKR 周期 ID（int64 类型）。当 `--level=objective` 时**必填**       |
+| `--objective-id` | 条件 | —      | 目标 ID。当 `--level=key-result` 时**必填**，用于定位父目标。           |
```
[the remark this answers is not in the document]
Comment (syh-cpdsss) [reply skills/lark-okr/references/lark-okr-reorder.md]: fixed

File: skills/lark-okr/references/lark-okr-indicator-update.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+| `--level`  | 是  | —      | 操作层级：`objective`（更新目标指标）\| `key-result`（更新 KR 指标） |
+| `--id`     | 是  | —      | 目标 ID 或 KR ID（int64 类型）                                       |
+| `--value`  | 是  | —      | 新的指标当前值（数字，范围：-99999999999 到 99999999999）              |
+| `--dry-run`| 否  | —      | 预览 API 调用而不实际执行                                            |
+| `--format` | 否  | `json` | 输出格式                                                             |
+
+## 工作流程
+
+1. 使用 `+cycle-list` 和 `+cycle-detail` 获取目标 ID 或 KR ID。
+2. 如需查看当前指标值，使用 `objective.indicators.list` 或 `key_result.indicators.list` 查询。
```
[the remark this answers is not in the document]
Comment (syh-cpdsss) [reply skills/lark-okr/references/lark-okr-indicator-update.md]: fixed

File: shortcuts/okr/okr_reorder_test.go [hunk 6]
```diff
@@ -0,0 +1,681 @@
...
+		t.Fatalf("expected typed error, got: %v", err)
+	}
+	validationErr, ok := err.(*errs.ValidationError)
+	if !ok {
+		t.Fatalf("expected ValidationError, got: %T", err)
+	}
+	if validationErr.Param != "--ops" {
+		t.Fatalf("expected param --ops, got %q", validationErr.Param)
+	}
+}
```
[the remark this answers is not in the document]
Comment (syh-cpdsss) [reply shortcuts/okr/okr_reorder_test.go:172]: fixed

**Review by zhangzq0 — APPROVED** [review]
